tests/unit/json_formatter_test.cc

```diff
@@ -112,6 +109,53 @@ SEASTAR_THREAD_TEST_CASE(test_stream_range_as_array) {
     auto buf = packets.release();
 
     sstring result(buf.front().get(), buf.front().size());
-    sstring expected = "[{\"subject\":\"1\",\"values\":[1]}, {\"subject\":\"2\",\"values\":[2]}, {\"subject\":\"3\",\"values\":[3]}]";
     BOOST_CHECK_EQUAL(expected, result);
 }
+
+
+SEASTAR_THREAD_TEST_CASE(test_stream_range_as_array) {
+    sstring expected = "[{\"subject\":\"1\",\"values\":[1]}, {\"subject\":\"2\",\"values\":[2]}, {\"subject\":\"3\",\"values\":[3]}]";
```

Comment:
nit, could use raw-string literal. like
```c++
sstring expected = R"([{"subject":"1","values":[1]},....)";
```

more readable this way.

Reply:
done